### PR TITLE
add Dropify@0.2.1 w/ npm auto update

### PR DESCRIPTION
**Pull request by onlineth**
https://github.com/cdnjs/cdnjs/issues/8349
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/JeremyFagis/dropify
 * Official website (optional, not the repository): http://jeremyfagis.github.io/dropify/
 * NPM package url (optional): https://www.npmjs.com/package/dropify
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 395
   - Count of watchers: 19
   - Count of forks: 60
 * NPM download stats (optional):
   - Downloads in the last day: 2
   - Downloads in the last week: 39
   - Downloads in the last month: 429

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/lib/Dropify/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    {
+      "basePath": "dist",
+      "files": [
+        "css/dropify*",
+        "js/*"
+      ]
+    }
+  ],
+  "license": "MIT"
+}
```
Comment (PeterDaveHello) [inline ajax/lib/Dropify/package.json]: Missing new line char here.

File: ajax/lib/Dropify/package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "author": "Jeremy FAGIS",
+  "homepage": "http://jeremyfagis.github.io/dropify/",
+  "npmName": "dropify",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "css/dropify*",
+        "js/*"
```
Comment (PeterDaveHello) [inline ajax/lib/Dropify/package.json]: I prefer to use `"**/*"` to grab all the files under dist folder, including the fonts.

Comment (PeterDaveHello) [inline ajax/lib/Dropify/package.json]: To exclude the demo css, maybe `"**/!(demo.css)"`

**Comment by PeterDaveHello** [comment]
@onlineth let's not directly drop the current PR to open a new one, you can update the PR to match the conventions and tests.

**Comment by PeterDaveHello** [comment]
@cdnjs/intern2 please help review this PR as your practice.

**Comment by onlineth** [comment]
@PeterDaveHello Alright, I'll make changes without dropping the PR. I've also added a new line and updated the npm auto-update per your request.

File: ajax/lib/Dropify/package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "author": "Jeremy FAGIS",
+  "homepage": "http://jeremyfagis.github.io/dropify/",
+  "npmName": "dropify",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*",
+        "**/!(demo.css)"
```
Comment (pvnr0082t) [inline ajax/lib/Dropify/package.json]: Let's combine these two into `"**/!(demo.css)"`

File: ajax/lib/Dropify/package.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  "name": "dropify",
+  "version": "0.2.1",
+  "filename": "js/dropify.min.js",
+  "description": "Override your input files with style.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/JeremyFagis/dropify.git"
+  },
+  "author": "Jeremy FAGIS",
+  "homepage": "http://jeremyfagis.github.io/dropify/",
```
Comment (kennynaoh) [inline ajax/lib/Dropify/package.json]: I think `https://jeremyfagis.github.io/dropify/` will be better.
Thanks!

**Comment by onlineth** [comment]
@pvnr0082t Just combined them (didn't know that could work, +1:)
@kennynaoh Edited the url for the homepage to use ssl

**Comment by PeterDaveHello** [comment]
@onlineth you'll need to add the fonts in the commit.

**Comment by onlineth** [comment]
@PeterDaveHello Added

**Comment by PeterDaveHello** [comment]
Wrong path as #8369, `ajax/lib` should be `ajax/libs`, please fix it.

**Comment by onlineth** [comment]
@PeterDaveHello Whoops, I'm on a sparse-checkout so that was the issue there, it should be good to go now.

**Comment by PeterDaveHello** [comment]
Thanks, and let's add some keywords:
https://github.com/JeremyFagis/dropify/blob/master/bower.json#L5-L10

Then we can merge this PR!

**Pull request merged**